cicd/gitlab/st/statusTracking.py

```diff
@@ -65,23 +65,21 @@ def parse_result(listOfTasks, checkPublication=False):
             task['pubSummary'] = '%d/%d/%d' % (failedPublications, published_in_transfersdb, published_in_dbs)
 
             if total_jobs > 0 and ('finished', total_jobs) in task['jobsPerStatus'].items():
-                if task['status'] == 'COMPLETED':
-                    result = 'TestPassed'
-                else:
-                    needToResubmit = True
                 if checkPublication and 'nopublication' not in task['taskName']:
-                    # remove probe jobs (job id of X-Y kind) if any from count
-                    jobsToPublish = total_jobs
-                    for job in task['jobs'].keys():
-                        if '-' in job:
-                            jobsToPublish -= 1
+                    # Remove probe jobs (job id of X-Y kind) from count
```

Comment:
I fail to see how this code gives different result to previous one. It should still give -4  for jobsToPublish. It is simply a bit/ easier to read.
BTW please let's stick to snakeCase convention.